actionmailbox/lib/action_mailbox/base.rb

```diff
@@ -78,14 +78,16 @@ def initialize(inbound_email)
     end
 
     def perform_processing # :nodoc:
-      track_status_of_inbound_email do
-        run_callbacks :process do
-          process
+      ActiveSupport::Notifications.instrument "process.action_mailbox", instrumentation_payload do
```

Comment:
Where is the documentation of this new event? It should be added here https://github.com/rails/rails/blob/main/guides/source/active_support_instrumentation.md#rails-framework-hooks

Reply:
Sorry @rafaelfranca - I wasn't aware of that documentation. I'll add it now.

Reply:
No problem. I thought we added in a later commit.